### PR TITLE
Update add-storage-space.md

### PR DESCRIPTION
**Pull request by msbemba**
I can see the option to add storage on A3 and A5 faculty at 0.17$ per GB. Made changes to the document accordingly. Please verify and approve.

Thanks
Sri

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by msbemba** [comment]
issue#5373

File: microsoft-365/commerce/add-storage-space.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ Office 365 Extra File Storage is available for the following subscriptions:
 - Office 365 Enterprise E3
 - Office 365 Enterprise E4
 - Office 365 Enterprise E5
+- Office 365 A3(faculty)
+- Office 365 A5(faculty)
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/add-storage-space.md]:
```suggestion
- Office 365 A3 (faculty)
- Office 365 A5 (faculty)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Pull request merged**